gax/src/main/java/com/google/api/gax/rpc/Callables.java

```diff
@@ -56,7 +56,10 @@ private Callables() {}
       ClientContext clientContext) {
 
     if (areRetriesDisabled(callSettings.getRetryableCodes())) {
-      return innerCallable;
```

Comment:
I think this (and below) could use a comment. Something like:
```
// When retries are disabled, the total timeout can be treated as the rpc timeout.
```

Reply:
done